### ISSUE TITLE
Inconsistent formatting in extension methods for tuples

### ISSUE THREAD

**Issue by CnSimonChan** [dotnet/roslyn#15570]
**Version Used**: 

Latest `master` branch

Visual Studio 2015 Update 3

**Steps to Reproduce**:

Suddenly there is an idea in my head: *Can I `await` on a tuple of multiple tasks, then get their results with deconstruction?* So I wrote this extension method:

```` C#
public static class AwaitTuples
{
    public static TaskAwaiter<(T1, T2)> GetAwaiter<T1, T2>(this(Task<T1> task1, Task<T2> task2) tasks)
    {
        return Task.Run(async () =>
        {
            await Task.WhenAll(tasks.task1, tasks.task2);
            return (tasks.task1.Result, tasks.task2.Result);
        }).GetAwaiter();
    }
}
````

However, when I invoke the *Format Document* command, I noticed one thing: **The space after `this` keyword is missing**

**Comment by HaloFour** [comment]
Formatting issues aside you might find the following implementation preferable since it doesn't require dispatching another `Task` onto the threadpool:

```cs
public static TaskAwaiter<(T1 result1, T2 result2)> GetAwaiter<T1, T2>(this (Task<T1> task1, Task<T2> task2) tasks) {
    async Task<(T1, T2)> join() {
       	await Task.WhenAll(tasks.task1, tasks.task2);
     	return (await tasks.task1, await tasks.task2);
    };
    return join().GetAwaiter();
}
```

**Comment by Pilchie** [comment]
Moving to the compiler for the support for new language features and tagging @jcouv.

**Comment by jcouv** [comment]
Note: this issue also reproes in a recent RC for dev15 (Version 15.0.26006.2 D15REL).

**Comment by jcouv** [comment]
Fixed by #15967
